en/docs/how-tos/adding-apis/importing-a-single-api-from-wso2-api-manager.md

```diff
@@ -54,6 +54,43 @@ Project 'marketing_project' is initialized successfully.
     -  Trust store location and password of WSO2 API Microgateway Toolkit - If you want to use the default value, click enter.
     ```
 
+    !!! note
```

Comment:
```suggestion
    !!! note - Importing APIs from APIM 4.x versions
```

Reply:
It's not possible to add a header to a note. Therefore, added this as a bold text inside the note from https://github.com/wso2/docs-mg/pull/184/commits/50d67d45206cb74d0620e3be7f91ee74c6d8bb38.